app/index.html

```diff
@@ -58,6 +58,10 @@ <h6 class="docs-header">What is this for?</h6>
             <label for="movie-data">MOVIES Slot</label>
             <textarea class="u-full-width text-area" id="movie-data" disabled></textarea>
           </div>
+          <div class="six columns">
+            <label for="moviegenre-data">MOVIEGENRES Slot</label>
+            <textarea class="u-full-width text-area" id="moviegenre-data" disabled></textarea>
+          </div>
         </div>
```

Comment:
These columns are like in Twitter's Bootstrap, if you've ever used it before.

Basically, they should add up to 12 in total.  So, since there were two before, you should change the existing two's class to "four columns" and make the new one "four columns" too.